### PR TITLE
Bump to latest and add pre release

### PR DESCRIPTION
**Pull request by xlight05**
## Purpose
$Subject
Related to : https://github.com/ballerina-platform/module-ballerinax-azure.functions/issues/535

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: ballerina/Ballerina.toml [hunk 1]
```diff
@@ -1,13 +1,13 @@
 [package]
 org = "ballerinax"
 name = "azure_functions"
-version = "3.0.0"
+version = "3.0.0-alpha.1"
```
Comment (anuruddhal) [inline ballerina/Ballerina.toml:4]: Why are we releasing an alpha version? Shall we use 3.0.0?

File: compiler-plugin/src/main/java/org/ballerinax/azurefunctions/AzureFunctionNameGenerator.java [hunk 2]
```diff
@@ -45,7 +45,8 @@ private String getFunctionName(String servicePath, FunctionDefinitionNode functi
                 String specialCharReplacedPathBlock = (((IdentifierToken) pathBlock).text()).replace("\\", "");
                 resourcePath.append("/").append(specialCharReplacedPathBlock);
             } else if (pathBlock.kind() == SyntaxKind.RESOURCE_PATH_SEGMENT_PARAM) {
-                Token token = ((ResourcePathParameterNode) pathBlock).paramName();
+                //TODO changed
```
Comment (anuruddhal) [inline compiler-plugin/src/main/java/org/ballerinax/azurefunctions/AzureFunctionNameGenerator.java:48]: Remove todo comment.

File: compiler-plugin/src/main/java/org/ballerinax/azurefunctions/service/http/HTTPTriggerBinding.java [hunk 3]
```diff
@@ -84,7 +84,8 @@ public List<FunctionContext> getBindings() {
                 if (pathBlock.kind() == SyntaxKind.RESOURCE_PATH_SEGMENT_PARAM) {
                     ResourcePathParameterNode pathParamNode = (ResourcePathParameterNode) pathBlock;
                     //TODO Handle optional
-                    resourcePath.append("/" + "{").append(pathParamNode.paramName().text()).append("}");
+                    //TODO changed
```
Comment (anuruddhal) [inline compiler-plugin/src/main/java/org/ballerinax/azurefunctions/service/http/HTTPTriggerBinding.java:87]: Remove todo comment.

File: compiler-plugin/src/main/java/org/ballerinax/azurefunctions/AzureFunctionNameGenerator.java (hunk 2 above)
Comment (xlight05) [reply compiler-plugin/src/main/java/org/ballerinax/azurefunctions/AzureFunctionNameGenerator.java]: Done.

File: compiler-plugin/src/main/java/org/ballerinax/azurefunctions/service/http/HTTPTriggerBinding.java (hunk 3 above)
Comment (xlight05) [reply compiler-plugin/src/main/java/org/ballerinax/azurefunctions/service/http/HTTPTriggerBinding.java]: Done.

**Pull request merged**